ballerina/tests/failover_connection_tests.bal

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerinax/activemq.driver as _;
+
+final Connection TEST_FAILOVER_CONNECTION = check new (
+    initialContextFactory = "org.apache.activemq.jndi.ActiveMQInitialContextFactory",
+    providerUrl = "failover:(tcp://localhost:61616)"
+);
+
+final Session FAILOVER_AUTO_ACK_SESSION = check createSession(AUTO_ACKNOWLEDGE);
```

Comment:
any reason to use snake case for non-constant variables?

Reply:
These variables are effectively constant for these tests.

Reply:
But agreed to your point. Fixed with [b3b9a72757de6a0a2068ca2a073d40e1cb46eb1b](https://github.com/ballerina-platform/module-ballerinax-java.jms/pull/135/commits/b3b9a72757de6a0a2068ca2a073d40e1cb46eb1b)

Reply:
@NipunaRanasinghe reverted the chage did due to build failure. Will do this in another PR as this convention has been used in other places in the codebase.

Reply:
+1